src/main/java/frc/robot/Robot.java

```diff
@@ -72,7 +64,8 @@ public void robotPeriodic() {
     // block in order for anything in the Command-based framework to work.
     CommandScheduler.getInstance().run();
 
-    Logger.recordOutput("Gyro/Rotation", robotContainer.gyro.getRotation2d());
+    // Logger.recordOutput("Gyro/Rotation", robotContainer.gyro.getRotation2d());
+    Logger.recordOutput("shooter", robotContainer.shooter.getShooterVelocity());
```

Comment:
Follow the convention for AdvantageKit fields. Use `Shooter/Top/Setpoint/Velocity` (what the code tells it to do) and `Shooter/Top/Measured/Velocity` (what's actually happening). They mean different things, so you'll need to add more. (Same goes for bottom shooter). It doesn't hurt to log more than we need.

Also, move this to the shooter subsystem. The only values that should be here are passive sensors that need to be logged when the robot is disabled. We want to isolate parts into their own subsystems as much as possible.